libs/langchain/langchain/document_loaders/parsers/__init__.py

```diff
@@ -34,6 +35,7 @@
     "PyMuPDFParser": "langchain_community.document_loaders.parsers.pdf",
     "PyPDFium2Parser": "langchain_community.document_loaders.parsers.pdf",
     "PyPDFParser": "langchain_community.document_loaders.parsers.pdf",
+    "ZeroxPDFParser": "langchain_community.document_loaders.parsers.pdf",
```

Comment:
could you remove this

Reply:
How we can import ZeroxPDFParser with `from langchain_community.document_loaders.parsers import ZeroxPDFParser` ?